test/battle/gimmick/dynamax.c

```diff
@@ -1431,6 +1431,7 @@ DOUBLE_BATTLE_TEST("(DYNAMAX) G-Max One Blow bypasses Max Guard for full damage"
 }
 
 // Bug Testing
+//  This test will fail if it's the first test a thread runs
```

Comment:
Any idea why?

Reply:
Nope. There are two other comments like that in this file

Reply:
Confirmed that it's a thing as of current master